salt/modules/openscap.py

```diff
@@ -1,110 +1,257 @@
+    KEY_RULES = {
+        "Supported specifications": {"split_at": ": "},
+        "Capabilities added by auto-loaded plugins": {"split_at": ": "},
+        "Paths": {"split_at": ": "},
+        "Inbuilt CPE names": {"split_at": " - "},
+    }
+
+    for line in _clean_version:
+        if line.startswith("===="):
+            _head = line.strip("=")[1:-1]
+            _resdict[_head] = {}
+            continue
+
+        if _head == "Supported OVAL objects and associated OpenSCAP probes":
+            if line.startswith("OVAL") or line.startswith("---"):
+                continue
+            _line = line.split(" ")
+            while "" in _line:
+                _line.remove("")
+            _resdict[_head][_line[0]] = {
+                "OVAL object": _line[1],
+                "OpenSCAP probe": _line[2],
+            }
+            continue
+
+        _key = line.split(KEY_RULES[_head]["split_at"])[0]
+        _value = line.split(KEY_RULES[_head]["split_at"])[1]
+        _resdict[_head][_key] = _value
+
+    _bin = _oscap_cmd().split("/")[-1]
+    short_version = {"{}".format(_bin): _prep_version[0].split(" ")[-1]}
+    if "full" in args:
+        return _resdict.update(short_version)
+
+    # Default return
+    return short_version
+
+
+def xccdf(file="", operation="eval", upload=True, **kwargs):
     """
     Run ``oscap xccdf`` commands on minions.
-    It uses cp.push_dir to upload the generated files to the salt master
-    in the master's minion files cachedir
+    It uses ``cp.push_dir`` to upload the generated files to the salt master.
     (defaults to ``/var/cache/salt/master/minions/minion-id/files``)
 
     It needs ``file_recv`` set to ``True`` in the master configuration file.
 
+    Args:
+        file (str): Target File to evaluate the system.
+        operation (str, optional): Operation of Module. Defaults to "eval".
+                    It will automatically add and set the parameters 'oval-results', 'report' and, 'results'.
+        upload (bool, optional): Upload results to Master. Defaults to True.
+
+    Returns:
+        dict: {
+            "success": _OSCAP_EXIT_CODES_MAP[_retcode],
+            "upload_dir": _upload_path,
+            "error": None (Kept in for compatibility reasons. It does not change!),
+            "returncode": _retcode
+            }
+
     CLI Example:
 
     .. code-block:: bash
+        salt '*' openscap.xccdf /usr/share/openscap/scap-yast2sec-xccdf.xml profile=default
+
+    All additional parameters need to either be set with a boolean or a string value.
+    E.g. the oscap '--fetch-remote-resources' parameter of 'xccdf eval' is just switching on the progress bar.
+    To enable it you have to set the parameter like this:
 
-        salt '*'  openscap.xccdf "eval --profile Default /usr/share/openscap/scap-yast2sec-xccdf.xml"
+    .. code-block:: bash
+        salt '*' openscap.xccdf /usr/share/openscap/scap-yast2sec-xccdf.xml profile=default fetch-remote-resources=true
```

Comment:
**suggestion**: add a blank line after  `.. code-block::`

Reply:
Done.

[This guide](https://docs.saltproject.io/en/master/topics/development/conventions/documentation.html#salt-docs) should provide some more guidance on what questions should be answered, when writing documentation.